payload-schemas/schemas/repository_dispatch.schema.json

```
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "repository_dispatch",
+
+  "type": "object",
+  "required": [
+    "action",
+    "branch",
+    "client_payload",
+    "repository",
+    "organization",
+    "sender",
+    "installation"
+  ],
+  "properties": {
+    "action": {
+      "type": "string"
+    },
+    "branch": {
+      "type": "string"
+    },
+    "client_payload": {
+      "type": "object",
+      "required": ["unit", "integration"],
+      "properties": {
+        "unit": {
+          "type": "boolean"
+        },
+        "integration": {
+          "type": "boolean"
+        }
+      },
+      "additionalProperties": false
```

`client_payload` is always different. The `unit` and `integration` properties are probably just coming from an example, we can remove them. And `"additionalProperties"` should be true
```suggestion
      "properties": {},
      "additionalProperties": true
```

---

Thanks, I missed that somehow. Fixed now.